examples/examples_nodejs_test.go

```diff
@@ -23,7 +23,6 @@ func TestAccRecordTs(t *testing.T) {
 func getJSBaseOptions(t *testing.T) integration.ProgramTestOptions {
 	base := getBaseOptions(t)
 	baseJS := base.With(integration.ProgramTestOptions{
-		RunUpdateTest: true,
```

Comment:
AccountID property is newly required since the baseline version so this needs to be disabled for now.